framework/sandstone.cpp

```diff
@@ -3777,8 +3783,14 @@ int main(int argc, char **argv)
                     logging_printf(LOG_LEVEL_QUIET, "# WARNING: test list is not empty while built-in test list provided.\n");
                 }
             }
-            for (auto& test : ordered_test_list) {
-                add_test(test_list, test);
+            builtin_test_list = get_test_list(builtin_test_list_name, &builtin_test_list_sz);
+            if (!builtin_test_list) {
+                logging_printf(LOG_LEVEL_QUIET,
+                        "# ERROR: no such list '%s'\n", builtin_test_list_name);
```

Comment:
Please say '--use-builtin-test-list' somewhere in this message so we can understand what the tool is complaining about when it prints the error.